src/status_im/ui/screens/chat/stickers/views.cljs

```diff
@@ -64,12 +64,16 @@
 
 (defn update-scroll-position [ref installed-packs selected-pack window-width]
   (when ref
-    (let [x (if (= selected-pack :recent)
-              0
-              (* (inc (some #(when (= selected-pack (:id (second %))) (first %))
-                            (map-indexed vector installed-packs)))
-                 window-width))]
-      (.scrollTo ref (clj->js {:x x :animated true})))))
+    ;; bug on Android
```

Comment:
Maybe you can provide a bit more detail as to why timeout is necessary on Android

Reply:
agreed need more explanation

Reply:
done